manual-ja.txt

```diff
@@ -17,7 +17,7 @@ AnkiDroidは，現在のところ，PC版のAnkiと合わせて使うことを
 
 == 更新履歴
 
-最近の更新履歴については，githubリポジトリの https://github.com/ankidroid/Anki-Android/blob/master/res/values/13-newfeatures.xml[release notes] を参照してください．
+最近の更新履歴については，https://ankidroid.org/docs/changelog.html[change log] を参照してください．
```

Comment:
The link hasn't be generated yet (because of lack of a space in front of the address?)

![image](https://cloud.githubusercontent.com/assets/10436072/5987665/66ce8aa8-a97b-11e4-90c2-35653d360d28.png)